tutorials/hyperspectral/Advanced_Spectral_Signatures.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Advanced Spectral Signatures in R"
+code1: HYPERSPECTRAL/Advanced_Spectral_Signatures.R
+contributors: null
+output:
+  html_document:
+    df_print: paged
+dateCreated: 2020-02-18 20:18:34
+description: Plot the spectral signatures of multiple different land cover types using
+  an interactive click-to-extract interface.
+estimatedTime: 0.5 Hours
+languagesTool: R
+dataProudct: null
+packagesLibraries: rhdf5, raster, plyr, reshape2, ggplot2
+syncID: db9715ca243944fabbe81031f2ed5cec
+authors: Donal O'Leary
+topics: hyperspectral, HDF5, remote-sensing
+tutorialSeries: null
+urlTitle: advanced-spectral-signatures-in-r
```

Comment:
change here to match new title (abbreviate is fine)

Reply:
How about "select-pixels-compare-spectral-signatures-r"